src/kernels/MIOpenPoolingBwd.cl

```diff
@@ -284,9 +284,8 @@ mloPoolingMaxBwd(const __global _FLOAT* top_df,
 
     barrier(CLK_LOCAL_MEM_FENCE);
     _FLOAT add_val;
-    int bt_y  = (y + lcl_id1 * MLO_POOLBWD_N_VERT_OUT_PIX);
-    int bt_x  = (x + lcl_id0 * MLO_POOLBWD_N_HORIZ_OUT_PIX);
-    int b_idx = bt_y * mlo_bot_width + bt_x;
```

Comment:
is "b_idx" not used anywhere and redundant?

Reply:
@muralinr it seems that we have met the same issue:
https://github.com/ROCmSoftwarePlatform/MIOpen/issues/1732

Reply:
Yes Jun. I looked at code.  'b_idx" is computed and it was not used inside code anywhere.  It's should be removed it to avoid compilation issues.